tests/e2e/cluster_creation_test.go

```diff
@@ -4,7 +4,11 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	CI "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
```

Comment:
This import should be removed. It is dup with the new import

Reply:
updated.